app/auth/screens/SignInScreen.tsx

```diff
@@ -0,0 +1,95 @@
+import { useState } from 'react';
+import { View, Text, TextInput, Pressable, ScrollView } from 'react-native';
+import { Link } from 'expo-router';
+import { useAuth } from '../context/AuthContext';
+import { Redirect } from 'expo-router';
+import { randomDirector } from '../../shared/utils/randomDirector';
+import { TabsPath } from '../../../constants/urls';
+
+const SignInScreen = () => {
+  const { signIn, user, isLoading } = useAuth();
+  const [email, setEmail] = useState<string | undefined>(undefined);
+  const [password, setPassword] = useState<string | undefined>(undefined);
+  const [error, setError] = useState<string | undefined>(undefined);
+
+  if (user) {
+    return <Redirect href={TabsPath} />;
+  }
+
+  const handleSignIn = async () => {
+    if (email && password) {
+      try {
+        setError(undefined);
+        await signIn(email, password);
+      } catch (error) {
+        setError('failed to sign in');
+        console.error(error);
+      }
+    } else {
+      setError('please provide both email and password');
+    }
+  };
+
+  return (
+    <ScrollView className="flex-1 bg-background">
+      <View className="container mx-auto">
+        <Text className="text-7xl text-center mt-20 text-text">cinesync</Text>
+      </View>
+
+      <View className="max-w-md mx-auto mt-5">
+        <View className="flex-row justify-center py-2">
+          <Text className="w-24 text-center text-text mt-3">email</Text>
+          <TextInput
+            className="pl-1 w-60 bg-secondary text-text rounded"
+            placeholder={randomDirector()}
+            placeholderTextColor="#999999"
+            value={email ?? undefined}
```

Comment:
Can we just past `email` since the whole point of making the types match was to avoid using `??`?